docs/sql/index.md

```diff
@@ -91,3 +93,56 @@ POST _opendistro/_sql/_explain
   "size": 50
 }
 ```
+
+### Syntax analysis exception
+
+
+If you get the following "Invalid SQL query" error:
+
+
+```json
+{
+  "reason": "Invalid SQL query",
+  "details": "Failed to parse query due to offending symbol [:] at: 'SELECT * FROM xxx WHERE xxx:' <--- HERE...
+    More details: Expecting tokens in {<EOF>, 'AND', 'BETWEEN', 'GROUP', 'HAVING', 'IN', 'IS', 'LIKE', 'LIMIT',
+    'NOT', 'OR', 'ORDER', 'REGEXP', '*', '/', '%', '+', '-', 'DIV', 'MOD', '=', '>', '<', '!',
+    '|', '&', '^', '.', DOT_ID}",
+  "type": "SyntaxAnalysisException"
+}
+```
+
+To resolve this error:
+
+1. Check if your syntax follows the MySQL grammar.
+1. If your syntax is correct, disable strict query analysis:
+
+```json
+PUT _cluster/settings
+{
+  "persistent" : {
+    "opendistro.sql.query.analysis.enabled" : false
+  }
+}'
+```
+
+Run the query again to see if it works.
+
+### Index mapping verification exception
+
+
+If you see the following verification exception:
+
+```json
+{
+  "error": {
+    "reason": "There was internal problem at backend",
+    "details": "When using multiple indices, the mappings must be identical.",
+    "type": "VerificationException"
+  },
+  "status": 503
+}
+```
+
+Make sure the index in your query is not an index pattern and it doesn't have multiple types.
```

Comment:
is not an index pattern and doesn't have multiple types.

Reply:
Done.